Applications/Utils/MeshEdit/ResetPropertiesInPolygonalRegion.cpp

```diff
@@ -0,0 +1,217 @@
+/*
+ * \date 2014-09-25
+ * \brief Reset material properties in meshes in a polygonal region.
+ */
+
+// TCLAP
+#include "tclap/CmdLine.h"
+
+// ThirdParty/logog
+#include "logog/include/logog.hpp"
+
+// BaseLib
+#include "BaseLib/LogogSimpleFormatter.h"
+
+// FileIO
+#include "FileIO/readMeshFromFile.h"
+#include "FileIO/writeMeshToFile.h"
+#include "FileIO/XmlIO/Boost/BoostXmlGmlInterface.h"
+
+// GeoLib
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/Polygon.h"
+#include "GeoLib/AnalyticalGeometry.h"
+
+// MathLib
+#include "MathLib/Vector3.h"
+
+// MeshLib
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Node.h"
+#include "MeshLib/Elements/Element.h"
+#include "MeshLib/MeshEditing/ElementValueModification.h"
+
+GeoLib::Polygon rotatePolygonToXY(GeoLib::Polygon const& polygon_in,
```

Comment:
why not move this funciton to GeoLib so that the code can be reused by other people? do you have any concerns?